src/models/ClassPotentialC2HDM.cpp

```diff
@@ -2895,8 +2895,6 @@ void Class_Potential_C2HDM::SetCurvatureArrays()
   YIJSL = MatrixXcd::Zero(NLepton, NLepton);
   YIJPL = MatrixXcd::Zero(NLepton, NLepton);
 
-  std::complex<double> II(0, 1);
```

Comment:
was this unused?

Reply:
It shadowed the same definition in SMparah.h